### ISSUE TITLE
Fix strict PHPCS issues

### ISSUE THREAD

**Issue by dartiss** [dartiss/pride-bar#6]
```
FILE: /Users/davidartiss/Documents/Github/pride-bar/pride-bar.php
--------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------
 76 | ERROR | The $text arg must be a single string literal, not "ucwords(
    |       | 'Pride Bar style' )".
--------------------------------------------------------------------------------
```

**Comment by dartiss** [comment]
I'm using new VIP standards now so this has changed BUT all others (including. auto fixes) are now resolved

**Comment by dartiss** [comment]
Fixed in #7
